hypertrace-core-graphql-attribute-store/src/main/java/org/hypertrace/core/graphql/attributes/AttributeModelTranslator.java

```
@@ -57,7 +80,33 @@ public Optional<AttributeModel> translate(AttributeMetadata attributeMetadata) {
     }
   }
 
-  @SuppressWarnings("unused")
+  public Optional<AttributeMetadata> translate(final AttributeModel attributeMetadata) {
+    try {
+      return Optional.of(
+          AttributeMetadata.newBuilder()
+              .setId(attributeMetadata.id())
+              .setScopeString(attributeMetadata.scope())
+              .setKey(attributeMetadata.key())
+              .setDisplayName(attributeMetadata.displayName())
+              .setValueKind(this.convertType(attributeMetadata.type()))
+              .setUnit(attributeMetadata.units())
+              .setOnlyAggregationsAllowed(attributeMetadata.onlySupportsGrouping())
+              .setType(
+                  attributeMetadata.onlySupportsAggregation()
+                      ? AttributeType.METRIC
+                      : AttributeType.ATTRIBUTE)
+              .addAllSupportedAggregations(
+                  this.convertMetricAggregationTypes(
+                      attributeMetadata.supportedMetricAggregationTypes()))
+              .setGroupable(attributeMetadata.groupable())
+              .setCustom(attributeMetadata.isCustom())
+              .build());
+    } catch (final Exception e) {
+      LOGGER.warn("Dropping attribute {} : {}", attributeMetadata.id(), e.getMessage());
```

On write, we want to fail an invalid request, not drop it.